tap_gitlab/utils.py

```diff
@@ -29,7 +30,7 @@ def wrapper(*args, **kwargs):
                 timer = threading.Timer(every, semaphore.release)
                 timer.setDaemon(True)  # allows the timer to be canceled on exit
                 timer.start()
-                return result
+            return result
```

Comment:
We should replace this with the ratelimit function in singer-python. See https://github.com/singer-io/singer-python/blob/master/singer/utils.py#L20